javascript/sources/Node.cc

```diff
@@ -331,9 +331,8 @@ Value Node::getPadding(int edge) const {
       YGNodeStyleGetPadding(m_node, static_cast<YGEdge>(edge)));
 }
 
-Value Node::getGap(int gutter, ) {
-  return Value::fromYGValue(
-      YGNodeStyleGetGap(m_node, static_cast<YGGutter>(gutter)));
+float Node::getGap(int gutter) {
+  return YGNodeStyleGetGap(m_node, static_cast<YGGutter>(gutter));
```

Comment:
I left these as `YGValues` in the hope that in the future, we add support for percentages. I just couldn't figure it out at the time

Reply:
this makes sense now, thanks for the clarification

but I have problems with types `YGNodeStyleGetGap` returns `float` that can't be converted with Value::fromYGValue

Reply:
I think that is better to fix `YGNodeStyleGetGap` first, it should return `YGValue`

I can't pick the unit for this type now (should it be undefined or auto or digit?)

Reply:
Ah ok - didn't realise the these functions were inconsistent. The unit is `YGUnitPoint`

Reply:
> I left these as YGValues in the hope that in the future, we add support for percentages. I just couldn't figure it out at the time

@jacobp100 I added in support for percentage gaps when I ported your gap changes to [Taffy](https://github.com/DioxusLabs/taffy) (a rust flexbox library originally based off of Yoga). Diff is [here](https://github.com/DioxusLabs/taffy/pull/248/files#diff-78a705bd422b6ec7ece906ca29973427ec10834766ca5bf1609459f0e4f539d5) if it helps you figure things out.

Reply:
That seems like a great help 🙂.

During H1 2023 I have some work on my plate to try to port some of the layout examples in WPT to Yoga compatible fixtures as well. Seems like a great resource for validating correctness of new layout props outside the context of browsers.